app/src/main/java/com/example/oddhours/ui/home/HomeAdapter.kt

```diff
@@ -59,6 +59,9 @@ class HomeAdapter(private var jobList: List<JobModel>, val context: Context, pri
     val month = c.get(Calendar.MONTH)
     val day = c.get(Calendar.DAY_OF_MONTH)
 
+    val daysInMonth = intArrayOf(31,28,31,30,31,30,31,31,30,31,30,31)
```

Comment:
have you considered leap year here? i.e. february will have 29 days instead of 28

Reply:
I definitely missed considering that. Will work on that!!